### PR TITLE
Deploy to Goerli and Optimism Goerli

### LINKED ISSUE #79: Update contracts to support other testnets
#### Description

- We want to support deploying to Optimism Goerli, Goerli and Arbitrum Goerli

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

- Add configuration for goerli and optimism goerli

Closes #79

### CODE REVIEW

**Comment by alexkeating** [comment]
Current dependencies on/for this PR:
* main
  * **PR #37** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting/37" target="_blank">Graphite</a> 
    * **PR #57** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting/57" target="_blank">Graphite</a> 
    * **PR #42** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting/42" target="_blank">Graphite</a> 
      * **PR #53** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting/53" target="_blank">Graphite</a> 
        * **PR #61** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting/61" target="_blank">Graphite</a> 
          * **PR #66** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting/66" target="_blank">Graphite</a> 
            * **PR #68** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting/68" target="_blank">Graphite</a> 
              * **PR #69** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting/69" target="_blank">Graphite</a> 
                * **PR #75** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting/75" target="_blank">Graphite</a> 
                  * **PR #80** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting/80" target="_blank">Graphite</a>  👈
    * **PR #39** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting/39" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting/80?utm_source=stack-comment).

File: test/Constants.sol [hunk 1]
```diff
@@ -38,11 +38,17 @@ contract BaseConstants is CommonBase {
...
+      wormholeChainId: 2,
+      wormholeRelayer: 0x28D8F1Be96f97C1387e94A53e00eCcFb4E75175a,
+      chainId: 5,
+      rpcUrl: vm.envOr("GOERLI_RPC_URL", string("https://rpc.ankr.com/eth_goerli"))
+    });
+    chainInfos[420] = ChainConfig({
+      wormholeChainId: 24,
+      wormholeRelayer: 0x01A957A525a5b7A72808bA9D10c389674E459891,
+      chainId: 420,
+      rpcUrl: vm.envOr("OPTIMISM_GOERLI_RPC_URL", string("https://optimism-goerli.publicnode.com"))
```
Comment (alexkeating) [inline test/Constants.sol:51]: Using these chains with the tests won't work because we need to update the underlying Wormhole SDK

Comment (wildmolasses) [reply test/Constants.sol:51]: makes sense; can this file be moved to the root dir? total nit, not merge blocking

File: test/Constants.sol [hunk 2]
```diff
@@ -38,11 +38,17 @@ contract BaseConstants is CommonBase {
...
-    chainInfos[42_161] = ChainConfig({
-      wormholeChainId: 23,
-      wormholeRelayer: 0x27428DD2d3DD32A4D7f7C497eAaa23130d894911,
-      chainId: 42_161,
-      rpcUrl: vm.envOr("ARBITRUM_RPC_URL", string("https://rpc.ankr.com/arbitrum"))
+    chainInfos[5] = ChainConfig({
+      wormholeChainId: 2,
+      wormholeRelayer: 0x28D8F1Be96f97C1387e94A53e00eCcFb4E75175a,
+      chainId: 5,
+      rpcUrl: vm.envOr("GOERLI_RPC_URL", string("https://rpc.ankr.com/eth_goerli"))
```
Comment (wildmolasses) [inline test/Constants.sol:45]: should these env vars be added to the env template as well?

**Review by wildmolasses — APPROVED** [review]
lgtm, a couple small things that shouldnt block merge

File: test/Constants.sol (hunk 1 above)
Comment (alexkeating) [reply test/Constants.sol:51]: By root do you mean source directly? Our at the same level as our gitignore. I can create an issue to move this around.

File: test/Constants.sol (hunk 2 above)
Comment (alexkeating) [reply test/Constants.sol:45]: https://github.com/ScopeLift/l2-flexible-voting/pull/80/commits/b60bd086113bd2af85ea0bbdefdf2806f5804a3d

**Review by apbendi — APPROVED** [review]
LGTM

**Pull request merged**